sdk/communication/azure-communication-chat/swagger/SWAGGER.md

```diff
@@ -17,7 +17,7 @@ autorest SWAGGER.md
 
 ```yaml
 tag: package-chat-2021-09-07
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/communication/data-plane/Chat/readme.md
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/8dbeba81f3a838cd4b7efd70234f29cc1cdc7374/specification/communication/data-plane/Chat/readme.md
```

Comment:
why did we change the commit hash?

Reply:
@AikoBB I made it in correspondence with other modalities where swagger is generated based on the specific commit and not on the branch.  Do I need to rollback this change and regenerate from swagger in `main`?

Reply:
It's up to the chat team, if they want to go with the main branch then it's better to revert as it was. @LuChen-Microsoft need your approval here as well :)